FRENCH/src/ch07-02-defining-modules-to-control-scope-and-privacy.md

```diff
@@ -0,0 +1,249 @@
+<!--
+## Defining Modules to Control Scope and Privacy
+-->
+
+## Définir des modules pour gérer la portée et la protection
+
+<!--
+In this section, we’ll talk about modules and other parts of the module system,
+namely *paths* that allow you to name items; the `use` keyword that brings a
+path into scope; and the `pub` keyword to make items public. We’ll also discuss
+the `as` keyword, external packages, and the glob operator. For now, let’s
+focus on modules!
+-->
+
+Dans cette section, nous allons aborder les modules et les autres outils du
+système de modules, à savoir les *chemins* qui nous permettent de nommer les
+éléments; l'utilisation du mot-clé `use` qui importe un chemin dans la portée;
+et le mot-clé `pub` que rend public les éléments. Nous verrons aussi le mot-clé
+`as`, les paquets externes, et l'opérateur glob. Pour commencer, penchons-nous
+sur les modules !
+
+<!--
+*Modules* let us organize code within a crate into groups for readability and
+easy reuse. Modules also control the *privacy* of items, which is whether an
+item can be used by outside code (*public*) or is an internal implementation
+detail and not available for outside use (*private*).
+-->
+
+Les *modules* nous permettent de regrouper le code d'une crate pour une
+meilleure lisibilité et pour la facilité de réutilisation. Les modules
+permettent aussi de gérer la *protection* des éléments, qui précise si un
+élément peut être utilisé à l'extérieur du code (*c'est public*) ou s'il est
+un constituant interne et n'est pas disponible pour une utilisation externe
+(*c'est privé*).
+
+<!--
+As an example, let’s write a library crate that provides the functionality of a
+restaurant. We’ll define the signatures of functions but leave their bodies
+empty to concentrate on the organization of the code, rather than actually
+implement a restaurant in code.
+-->
+
+Voici un exemple, écrivons une crate de bibliothèque qui permet de simuler un
+restaurant. Nous allons définir les signatures des fonctions mais nous allons
+laisser leurs corps vides pour nous concentrer sur l'organisation du code,
+plutôt que de coder pour de vrai un restaurant.
+
+<!--
+In the restaurant industry, some parts of a restaurant are referred to as
+*front of house* and others as *back of house*. Front of house is where
+customers are; this is where hosts seat customers, servers take orders and
+payment, and bartenders make drinks. Back of house is where the chefs and cooks
+work in the kitchen, dishwashers clean up, and managers do administrative work.
+-->
+
+Dans le secteur de la restauration, certaines parties d'un restaurant sont
+assimilés à la *salle à manger* et d'autres *aux cuisines*. La partie salle à
+manger est l'endroit où se trouvent les clients; c'est l'endroit où les hôtes
+installent les clients, où les serveurs prennent les commandes et encaissent les
+clients, et où les barmans préparent des boissons. Dans la partie cuisines, nous
```

Comment:
`barman` feels very... western 🤠... but not sure it fits very well. This is very minor of course, meaning is clear

Reply:
Funny, but I don't find another word to translate it 😄
[linguee](https://www.linguee.com/english-french/search?query=bartender&source=auto)/deepl, [FranceTerme](http://www.culture.fr/franceterme/result?francetermeSearchTerme=bartender&francetermeSearchDomaine=0&francetermeSearchSubmit=rechercher&action=search), [wikipedia](https://fr.wikipedia.org/wiki/Barman) and [GDT ](http://www.granddictionnaire.com/ficheOqlf.aspx?Id_Fiche=26529044) doesn't find another way to translate it except ... `serveur` 🙄

Reply:
I do think `serveur` is fine, but you can keep `barman`, it might make some laugh which is a good thing imo 😄